docs/sql-ref-datatypes.md

```diff
@@ -194,7 +194,7 @@ The following table shows the type names as well as aliases used in Spark SQL pa
 |**DecimalType**|DECIMAL, DEC, NUMERIC|
 |**CalendarIntervalType**|INTERVAL|
 |**ArrayType**|ARRAY\<element_type>|
-|**StructType**|STRUCT<field1_name: field1_type, field2_name: field2_type, ...>|
+|**StructType**|STRUCT<field1_name field1_type, field2_name field2_type, ...>|
```

Comment:
Instead of removing `:`, how about writing `Note that : is optional` like https://github.com/apache/spark/blame/master/docs/sql-ref-datatypes.md#L172 ?